sdk/python/pulumi_kubernetes/yaml.py

```diff
@@ -463,6 +463,13 @@ def _parse_yaml_object(
             lambda identifier: f"{resource_prefix}-{identifier}")
 
     gvk = f"{api_version}/{kind}"
+
+    if gvk == "v1/Secret":
+        # Manually mark these fields as secret to avoid leaking sensitive values from raw YAML.
+        for key in ['data', 'string_data']:
```

Comment:
Curious - Are these the only sensitive input fields in the entire Kubernetes API?

Reply:
Those are the only ones I can think of that would ~always be sensitive.